### DISCUSSION TITLE
[值得关注] ROSS Index: The fastest-growing open-source startups

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#122] [2️⃣ 聊项目]
在 coss.community 上看到的一篇文章：[Real-time Open Source Indexes: Databases, Headless CMSs and Static Site Generators](https://www.coss.community/bogdansemenov/real-time-open-source-indexes-databases-headless-cmss-and-static-site-generators-15bp)

一个叫做 Runa Capital 发布的一个季度性的开源指数报告：**ROSS Index** (https://runacap.com/ross-index/)

![image](https://user-images.githubusercontent.com/15010826/177887017-2181655c-c801-4c19-b9e1-80d72fc97ac0.png)

简直是为我们定制的，在他们的这个基础之上，插入我们的活跃度指标与影响力指标，应该立即就可能看到更多有洞见的东西，特别是和投资端的数据联系起来的时候。他们其实也在不断更新与迭代[相关指标](https://medium.com/runacapital/introducing-real-time-open-source-indexes-databases-headless-cmss-and-static-site-generators-5b53cbf87188)，希望能够看的更深入，同时还推出了一些有意思的工具，类似：[Awesome list](https://github.com/RunaCapital/awesome-oss-alternatives)，[Growth Benchmarks](https://medium.com/runacapital/open-source-growth-benchmarks-and-the-20-fastest-growing-oss-startups-d3556a669fe6)，[Badges](https://runacap.com/ross-index/badges/)，都有很好的参考价值~

Runa Capital 在 medium 上的主页参见：https://medium.com/runacapital

非常值得更多的挖掘~

其中还有个叫做：[**Open-Source Database Index**](https://runacap.com/data/db-index/) 的东西，聚焦数据库方向的，对我们来说同样是个很好的参考，特别是对 @birdflyi

![image](https://user-images.githubusercontent.com/15010826/177887952-6b392b2c-b0bb-4f02-80dc-4e2a6cd1b45d.png)
 
同样，把这套东西复制到国内，结合”[中国开源项目商业公司](https://gitcode.net/open-source-lab/List-of-Chinese-Open-Source-Project-Financing)“清单，立马就可以出来了~

![image](https://user-images.githubusercontent.com/15010826/177889521-f50f9a51-74cc-4e13-9a7a-0dbf04ad5de3.png)

这两天看了下 [OSS Capital](https://github.com/orgs/X-lab2017/discussions/120) 的数据，受冲击很大，COSS 的[**万亿级市场**](https://www.youtube.com/watch?v=TUOGs_PdeYE)即将到来~

![image](https://user-images.githubusercontent.com/15010826/177890688-191960bc-2903-4858-8198-d3098e31b173.png)

💡 通过商业思维连接一切~

**Comment by will-ww** [comment]
如果能够找到一个感兴趣的头部投资机构，完全可以联合发布类似的，多样化的、周期性价值报告~

**Reply by wangleo61** [reply]
找雪哥推荐哈

**Comment by AliceCodeZhang** [comment]
好奇上面的图COSS里面有Gitlab 为什么没有Github

**Comment by wangleo61** [comment]
张老师，貌似Github本身不开源

发件人: Yanbin ***@***.***>
发送时间: 2022年7月8日 9:33
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [X-lab2017/open-wonderland] [值得关注] ROSS Index: The fastest-growing open-source startups (Discussion #122)


好奇上面的图COSS里面有Gitlab 为什么没有Github

**Comment by will-ww** [comment]
再补充几个 **[Open Source Benchmarks](https://medium.com/runacapital/open-source-growth-benchmarks-extention-the-ross-index-and-the-fastest-growing-startups-in-q3-2020-7aee7fa7eed7)**：

- [COSS Index](https://docs.google.com/spreadsheets/d/17nKMpi_Dh5slCqzLSFBoWMxNvWiwt2R-t4e_l7LPLhU/edit#gid=0)
- [BOSS Index](https://techcrunch.com/2017/04/07/tracking-the-explosive-growth-of-open-source-software/)
- [Open100](https://www.accel.com/noteworthy/the-future-of-open-source-launching-the-open100)

**Reply by bifenglin** [reply]
这个benchmark是怎么定义的。。。我看是OSS的一些数据统计分析和项目排名。

**Reply by will-ww** [reply]
> 这个benchmark是怎么定义的。。。我看是OSS的一些数据统计分析和项目排名。

嗯，你可以抽空看看，就是些统计指标算的，在 Tbench 的论文中，这种就是属于“最佳实践”类型的 benchmark 了~

**Comment by will-ww** [comment]
今日好图：**[Open Source Is Eating Software FASTER than Software Is Eating The World](https://www.coss.community/cossc/open-source-is-eating-software-faster-than-software-is-eating-the-world-3b01)**

![image](https://user-images.githubusercontent.com/15010826/177966161-0c551ece-8c1c-451d-91d2-9b112441fc5d.png)

**Comment by wangleo61** [comment]
这条曲线会交汇么 ：）

**Reply by will-ww** [reply]
Sure~

- Open Core: Open source eating software from the inside-out. Closed Core: Open source eating software from the outside-in.
- Open source alternatives will soon be over. They will become defaults, not alternatives.

**Comment by will-ww** [comment]
继续连接两类数据：就业市场，投资市场

**就业市场数据：**
- https://www.trueup.io/
- 有专门的 Open source 板块：https://www.trueup.io/open-source
- 该工具本身也是非常值得研究

![image](https://user-images.githubusercontent.com/15010826/178082491-b2cae212-061b-42d5-ba20-9aeb94093223.png)

**投资市场数据：**
- https://www.koyfin.com/
- 还不会玩，COSS 周报每周都会用这个数据源来看整个 COSS 的行情
- 非常强大、有免费基础版本、非常值得探索

![image](https://user-images.githubusercontent.com/15010826/178082557-041f18b2-9cf2-4651-8e61-af435d679ac0.png)

![image](https://user-images.githubusercontent.com/15010826/178082828-d222bc3c-9932-48de-ab30-31a4a410c2bb.png)

**Comment by will-ww** [comment]
关于COSSI的信息，可以参考这篇文章：https://www.coss.community/jj/the-100-billion-commercial-oss-index-cossi-511h

Here is some commentary on the data tracked in COSSI:

- **Company Name** (pretty important detail)
- **Year Founded** (looking at this for all the companies offers some rough insight into how long — on average — it takes to reach a baseline level of maturity)
- **Total amount of VC money raised** (exactly how capital-intensive is it to build a “successful” COSS company?)
- **Total # of Employees** (how many people does it take to reach $100M+ in commercial OSS revenue?)
- **Estimated Revenue** (emphasis on estimated, but more on this soon)
- **Estimated Valuation** (even more of a grey area and valuations are meaningless until exits happen, but this is fun to compute, even heuristically)
- **Public or Private** (is the company stock liquid on a public exchange or is it private?)
- **Exit Outcome Size** (how much the company exited at, if they did?)
- **Exit Date** (when did they exit, if they did?)
- **Technology Area** (what layer in the evolving tech stack does this company predominantly serve?)
- **Monetization / Business Model** (how does the company make money?)
- **Core Project** (name and link to the core OSS project in question?)
- **OSS License** (what is the OSS license of the core OSS project the company relies on? if many projects, what is the primary/dominant license?)
- **Foundation-Based** (is the core project in a foundation or not?)

Data sources:
- **LinkedIn for employee count** (very accurate)
- **Talking with VCs and founders for revenue and valuation #s** (when I can’t/don’t engage these sources, I have a heuristic that is quite accurate)
- **Crunchbase for funding** (which is usually conservative since CB omits seed/A/B extensions and other notes that are unreported)
